apmpackage/gendocs.go

```diff
@@ -0,0 +1,116 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package apmpackage
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+	"sort"
+	"strings"
+	"text/template"
+)
+
+func GenerateDocs(inputFields map[string]Fields, version string) {
```

Comment:
Unless you plan to call this from some other code in the near future, can you please move everything into `apmpackage/cmd/gen-package`, and unexport all symbols?

Reply:
awful excuse i guess, but i went this way because my IDE debugger doesn't like functions in `main` package 🤷